docroot/themes/humsci/humsci_basic/src/scss/elements/_base.scss

```diff
@@ -68,3 +68,26 @@ h6:not([class]) {
 p:not([class]) {
   @include hb-body--small;
 }
+
+
+// Hide the FontAwesome icon that is configured by default in
+// config/default/extlink.settings.yml
+// and replace with different icon
```

Comment:
Great solution for matching the designs while maintaining platform-wide compatibility with the extlink module. 🏆

Reply:
ugh apparently not... we will need to change the color of this icon from time to time, and on hover. I'm adding that issue to another card here: https://sparkbox.atlassian.net/browse/STN-136